backend/api/Helpers/Logging/LoggerExtension.cs

```diff
@@ -1,11 +1,13 @@
-﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration;
 using System;
 using System.Reflection;
 using Microsoft.Extensions.DependencyInjection;
 using Serilog;
+using System.Diagnostics.CodeAnalysis;
 
 namespace Pims.Api.Helpers.Logging
 {
+    [ExcludeFromCodeCoverage]
```

Comment:
All of these exclusions are based on my personal opinion of files that we don't need to test, generally because they are some kind of boilerplate configuration file.